app/recipe/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from recipe.models import Recipe
+
+
+class RecipeSerializer(serializers.ModelSerializer):
+    """Serializer for recipe objects"""
+
+    class Meta:
+        model = Recipe
+        fields = ('id', 'name', 'description')
+        read_only_fields = ('id',)
+
+
+class RecipeDetailSerializer(RecipeSerializer):
```

Comment:
We will want all endpoints (list and detail) to return the full recipe with ingredients. Do you think we will need two separated serializers?

Reply:
I dont' think so. I just realized in both scenarios we return the full recipe with ingredients. Fixed